doc/llvm_obfuscation_tutorial.rst

```diff
@@ -150,7 +162,7 @@ This could be greatly enhanced using `dominators <http://llvm.org/docs/doxygen/h
 Do or do not there is no... test
 ********************************
 
-I am sure that your are eager to compile and run this empty pass. Thanks to the files provided in the Quarkslab git it's actually quite easy.
```

Comment:
On Mon, Feb 02, 2015 at 05:55:03AM -0800, Merlini Adrien wrote:
> In doc/llvm_obfuscation_tutorial.rst:
> > @@ -150,7 +162,7 @@ This could be greatly enhanced using `dominators <http://
> llvm.org/docs/doxygen/h
> >  Do or do not there is no... test
> >  ********************************
> >
> > -I am sure that your are eager to compile and run this empty pass. Thanks to
> the files provided in the Quarkslab git it's actually quite easy.
> Tu veux pas faire de la pub pour Quarkslab? :O

It will published on the quarkslab blog and git, no need to state it
again here